skia/viewer/src/main.cpp

```diff
@@ -104,8 +106,15 @@ void initStateMachine(int index) {
     assert(fileBytes.size() != 0);
     auto file = rive::File::import(rive::toSpan(fileBytes), &skiaFactory);
     if (!file) {
-        fileBytes.clear();
         fprintf(stderr, "failed to import file\n");
+
+        auto data = SkData::MakeWithCopy(fileBytes.data(), fileBytes.size());
+        gImage = SkImage::MakeFromEncoded(data);
+        if (gImage) {
+            fprintf(stderr, "interpreted as image\n");
+        }
+
+        fileBytes.clear();
```

Comment:
Should we also set currentScene to nullptr? Otherwise I think this workflow won't work:

- drag .riv, displays as expected
- drag in a .png, will keep showing previous scene loaded from .riv

Unless I'm missing something...

Reply:
done